### DISCUSSION TITLE
如何在平台中为上传的文件生成一个下载链接？

### DISCUSSION THREAD

**Discussion by wuzhi-dev** [flagify-com/OctoMation#56] [General]
如何在平台中为上传的文件生成一个下载链接？

**Comment by wzfukui** [comment]
很抱歉，应用SDK生成文件链接依赖API，社区版暂未开放API。我们本周会发布社区版迭代的Roadmap，也在争取加快把大家关注的不同功能点尽快开放出来。因为企业版使用的一些代码和组件有特别授权，研发团队正在做社区移植，争取早日实现。
在暂时无法实现的情况下，可以考虑通过OSS存储后拿到下载链接的方式处置里。
